src/SSW.SophieBot/SSWSophieBot/dialogs/GetAvailableExpertsByLocationDialog/language-generation/en-us/GetAvailableExpertsByLocationDialog.en-us.lg

```diff
@@ -137,7 +137,7 @@
 
 # FreeDays(client, freeDays)
 -```
-${freeDays}/28 days
+${freeDays}/20 days
```

Comment:
How about just saying `XX days`? In the column header, we already have the time duration.

Reply:
I think having an explicit statement of total days would be better. Otherwise we would need to change the column header to 'Availability for next 20 days'